src/models/menu.entity.ts

```diff
@@ -27,4 +28,7 @@ export default class Dish {
 
     @Column({nullable: false})
     dishCategory: dishCategory
+
+    @OneToMany(() => CartPosition, cart => cart.dish)
+    cartPositions?: CartPosition[]
```

Comment:
Usually ORMs don't require you to define all possible fields and relations (curious if typeorm supports it), so if you know that you would never need some property in an entity, then you could skip it. E.g. most likely you would never want to find all orders for a specific product.